gamestonk_terminal/portfolio/portfolio_optimization/optimizer_view.py

```diff
@@ -345,6 +352,17 @@ def display_ef(stocks: List[str], period: str = "3mo", n_portfolios: int = 300):
         verbose=True, risk_free_rate=rfrate
     )
     ax.scatter(std_sharpe, ret_sharpe, marker="*", s=100, c="r", label="Max Sharpe")
+    # Add risk free line
+    if risk_free:
+        y = max(rets)
+        b = get_rf()
+        m = (ret_sharpe - b) / std_sharpe
+        x = (y - b) / m
+        ax.set_xlim(xmin=0)
```

Comment:
Personal opinion, but I don't think it looks as good with the xlim going to 0.  I would keep it to either like .95 of min(max_std) or just subtract a few percent from the min.

Reply:
The rf line starts at x=0. Shouldn't we show the entire line?

Reply:
I guess.  I just did an example and the volatility axis I saw was from .2 to .3, so extending to 0 looked weird

Reply:
It should always be at 0 since the risk-free asset has no volatility. Was this including -r? And which ticker?

Reply:
yeah.  My comment was that the ef (with no -r) went from like .2 to .28 and then the plot looked "squished" for the ef (fig here).  This was aapl,sbux , `ef -p 1y -r`.  Also you should add a label for the legend
![Figure_1](https://user-images.githubusercontent.com/18151143/136039572-c635df14-8532-4fbd-8341-ee54a6d242b5.png)
.

Reply:
Ooh yeah thats not good. Should I just remove the xmin altogether?

Reply:
Well without it, it will still go all the way down to zero, so as I mentioned earlier, just set it to be a little less than the min volatility value

Reply:
.95 looked weird when there was a deep negative stock like GME. So I switched it to .8 which was a lot better. Graph looks a lot better overall thanks for the input!